.github/publish-profiling-base-petclinic.yml

```diff
@@ -0,0 +1,25 @@
+name: publish-profiling-base-petclinic-image
+on: workflow_dispatch
```

Comment:
Why only manually? You can trigger it whenever image's source code changes

Reply:
I don't expect it to change often, and having it triggered by changes is likely to produce a lot of unnecessary images and take up build time, no?